games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.Artsoft-rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+    major  = @MAJOR@,
+    middle = @MIDDLE@,
+    minor  = @MINOR@,
+	
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0,
+
```

Comment:
@Begasus, there's something strange about _internal_. Why do we put it after _variety_ and not after _minor_ in the rdef file?
In other words, are you sure we can use it the way you suggest?